plugin.video.tweakers/addon.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon 
+	id="plugin.video.tweakers" 
+	name="Tweakers" 
+	version="1.1.2"
+	provider-name="Skipmode A1,Amelandbor,ninovanhooff">
+  <requires>
+    <import addon="xbmc.python"                 version="2.14.0"/>
```

Comment:
Ack, it should have been gotham, my bad. I'm going to close this PR now and do a gotham PR in the future.